### DISCUSSION TITLE
Modular: Starting Point for Apps Using Laravel, Vue.js, Tailwind & Inertia.js

### DISCUSSION THREAD

**Discussion by daniel-cintra** [HassanZahirnia/laravel-package-ocean#177] [Package Suggestions]
Modular is a project crafted to streamline the web application development using the VILT Stack (Vue 3, Inertia, Laravel, and Tailwind). It serves as a well-documented foundation for kickstarting your next application, offering a harmonious set of tools that synergize perfectly.

**Repository:**
https://github.com/daniel-cintra/modular

**Documentation:**
https://docs.ismodular.com/

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!

**Reply by daniel-cintra** [reply]
Thank you very much Hassan!
